### DISCUSSION TITLE
[Feature] Add sorting feature for shared albums shown in "Sharing"

### DISCUSSION THREAD

**Discussion by C-Otto** [immich-app/immich#11842] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The albums showed in "Sharing" are not sorted, nor can they be sorted. I'd love to be able to have the same sorting features as on the regular "Albums" page.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7025 ;)
